umh.docs.umh.app/content/en/docs/features/data-connectivity-node-red/index.md

```diff
@@ -3,31 +3,78 @@ title = "Data connectivity with Node-RED"
 menuTitle = "Data connectivity with Node-RED"
 description = ""
 weight = 4
-draft = true
+draft = false
 +++
 
-Introduction sentence
+One feature of the UMH is to connect devices on the shopfloor such as PLCs, Quality Stations or MES / ERP systems with the Unified Namespace using Node-RED. 
+Node-RED has a large library of nodes, which lets you connect various protocols. It also has a user-friendly UI
+with little code, making it easy to configure the desired nodes.
+
+![](/images/features/ifm-retrofitting/noderedDifferentProtocols.png?width=60%)
 
 ## When should I use it?
 
+Sometimes it is necessary to connect a lot of different protocols (e.g Siemens-S7, OPC-UA, Serial, ...) and node-RED can be a maintainable
```

Comment:
Avoid latin phrases
https://umh.docs.umh.app/docs/development/contribute/documentation/style/style-guide/#avoid-latin-phrases

Reply:
dont get that point. I didnt use "We"  \_(*.*)_/

Reply:
I was talking about the "e.g", but it's too late now I guess